src/library/helpers/portal-object-helpers.js

```diff
@@ -1,19 +1,26 @@
-var filterDiv = null
+let filterDiv = null
 
-var shortenText = function (text, length) {
+const shortenText = (text, length, breakOnSpace) => {
+  let shortenedText = text
   length = (typeof length !== 'undefined') ? length : 350
   if (!text) {
     return ''
   } else {
     if (text.length > length) {
-      return text.substring(0, length - 1) + '…'
+      if (breakOnSpace !== 'undefined') {
```

Comment:
This isn't the best approach because someone might pass `false` for the break on space argument.

@pjanik probably has a better solution but I'd suggest a simple:
`if (breakOnSpace) {`
That should be true for any value other than: undefined, '', 0, [], null, NaN, or false

Reply:
Your suggestion is fine, but as a point of interest, the empty array ([]) is not falsy.

Reply:
Thanks for catching that.

Reply:
Also, `!== 'undefined'` seems a bit suspicious. I'd rather expect `!== undefined` or `typeof ... !== 'undefined'`. But yea, personally I'd just check for falsy value as Scott suggested.

Reply:
Whoops -- thanks for catching that! I originally used `if (breakOnSpace)`, but then experimented with letting the variable value be the type of character you wanted to break on. I decided there wasn't much use in that, but then overlooked removing that part of the experiment.

I've fixed that and also made the JSX conversion.